### PR TITLE
modules/fontconfig/ultimate: preset remove warning

### PR DESCRIPTION
**Pull request by Profpatsch**

### CODE REVIEW

**Comment by Profpatsch** [comment]
Should probably be backported? Not sure if the changes are already in 17.03.

**Comment by vcunat** [comment]
No, 17.03 is using the old freetype and the update isn't suitable for backporting, I believe.

File: nixos/modules/rename.nix [hunk 1]
```diff
@@ -204,5 +204,6 @@ with lib;
       "Set the option `services.xserver.displayManager.sddm.package' instead.")
     (mkRemovedOptionModule [ "fonts" "fontconfig" "forceAutohint" ] "")
     (mkRemovedOptionModule [ "fonts" "fontconfig" "renderMonoTTFAsBitmap" ] "")
+    (mkRemovedOptionModule [ "fonts" "fontconfig" "ultimate" "preset" ] "See nixpkgs commit 65592837b6e62fb555d6e8c891f347428886c4f2");
```
Comment (vcunat) [inline nixos/modules/rename.nix]: The semicolon here seems wrong.

Comment (Profpatsch) [inline nixos/modules/rename.nix]: Crap, didn’t test it because I thought the change was so trivial. As always the wrong decision.

Comment (Profpatsch) [inline nixos/modules/rename.nix]: fixd

**Comment by ttuegel** [comment]
I did not intend to remove this option. It is restored in master now.

**Pull request closed without merging**